build.py

```diff
@@ -359,6 +359,37 @@ def get_path(self):
 
 Project.add(Tool_nuget())
 
+class Tool_perl(Tool):
+    def __init__(self):
+        Tool.__init__(self,
+            'perl',
+            archive_url = 'https://dl.hexchat.net/misc/perl/perl-5.20.0-x64.tar.xz',

```

Comment:
Yes, without a problem. I use this because I cannot find a 'simple' tar/zip perl to download and I used the hexchat server because it was indicated in the wingtk readme.md.

You're not talking of putting the file in the repository, right ? Just to have it in wingtk/xxx_something_perl.xv and download that when needed ? How we can make it ?